src/getCoinCombination.test.js

```diff
@@ -1,8 +1,26 @@
+'use strict';
+
 describe('getCoinCombination', () => {
   const { getCoinCombination } = require('./getCoinCombination');
 
   it(`should be declared`, () => {
```

Comment:
How about adding test like 'should return an array with zeros if cents = 0' and test where all array's indexes are more than 0 
cents = 41  => [1,1,1,1]

Reply:
yes,thanks